src/immer.js

```diff
@@ -299,6 +304,8 @@ export class Immer {
 			// Search new objects for unfinalized drafts. Frozen objects should never contain drafts.
 			else if (isDraftable(value) && !Object.isFrozen(value)) {
 				each(value, finalizeProperty)
+				// for externally incoming object trees, we wan't to make sure they're frozen automatically
```

Comment:
```suggestion
				// Freeze new objects when "autoFreeze" is enabled.
```

Reply:
Removed the comment, didn't really ad anything